web/templates/settings/display_settings.hbs

```diff
@@ -30,10 +30,10 @@
         </div>
     </div>
 
-    <div class="emoji-display-settings {{#if for_realm_settings}}settings-subsection-parent{{else}}subsection-parent{{/if}}">
+    <div class="emoji-preferences {{#if for_realm_settings}}settings-subsection-parent{{else}}subsection-parent{{/if}}">
         <div class="subsection-header">
             <h3 class="light">{{t "Emoji" }}</h3>
-            {{> settings_save_discard_widget section_name="emoji-display-settings" show_only_indicator=(not for_realm_settings) }}
+            {{> settings_save_discard_widget section_name="emoji-preferences" show_only_indicator=(not for_realm_settings) }}
```

Comment:
This needs to be in the previous commit -- since the only thing this `section_name` parameter does is generate a `<div class="alert-notification {{section_name}}-status"></div>`

Reply:
Thank you for your feedback, and I'll definitely keep these considerations in mind for future contributions. Thanks again!